### DISCUSSION TITLE
nikke助手打开没反应+1

### DISCUSSION THREAD

**Discussion by mengyuyou** [KIvoy/nikke-assistant#3] [Q&A]
老哥，这是我的app_log，也是打不开界面，但是在任务管理器可以看到在运行，连着点几次还能再在任务管理器同时看到几个在运行的，只能先直接结束任务退出，大佬麻烦看一下啦
[Uploading app_log.txt…]()

**Comment by KIvoy** [comment]
老哥可以下一下v1.1版本试一下， 那里增加了最新的报错以及多屏支持。 如果还是不行可以把app_error, app_log 和app_output都发上来看看， 应该能找到问题。
